docker/entrypoint.sh

```diff
@@ -53,18 +52,20 @@ function step_ca_init () {
         generate_password > "${STEPPATH}/password"
         generate_password > "${STEPPATH}/provisioner_password"
     fi
-    if [ -n "${DOCKER_STEPCA_INIT_SSH}" ]; then
+    if [ "${DOCKER_STEPCA_INIT_SSH}" == "true" ]; then
         setup_args=("${setup_args[@]}" --ssh)
     fi
-    if [ -n "${DOCKER_STEPCA_INIT_ACME}" ]; then
+    if [ "${DOCKER_STEPCA_INIT_ACME}" == "true" ]; then
         setup_args=("${setup_args[@]}" --acme)
     fi
-    if [ -n "${DOCKER_STEPCA_INIT_REMOTE_MANAGEMENT}" ]; then
-        setup_args=("${setup_args[@]}" --remote-management)
+    if [ "${DOCKER_STEPCA_INIT_REMOTE_MANAGEMENT}" == "true" ]; then
```

Comment:
I'm fine with having to set them to `true`. It might be an option to have it take multiple truthy/falsy values that then get used, but that could be done later.